manifests/cf-manifest/scripts/generate-cf-secrets.sh

```diff
@@ -37,7 +35,6 @@ cat <<EOF
     uaa_clients_cloud_controller_username_lookup_secret: $(simplepass)
     uaa_clients_gorouter_secret: $(simplepass)
     uaa_clients_firehose_password: $(simplepass)
-    uaa_admin_password: $(simplepass)
```

Comment:
I'm reluctant to do so unless/until we have a means of enforcing the order for subsequent changes.